dashboard/app/helpers/lti_helper.rb

```diff
@@ -0,0 +1,119 @@
+
+module LtiHelper
+
+  def windows_complex_password
```

Comment:
The user should never see this password. Looking at the User model the validation constrains it to 6..128 characters in length but I guess I don't see the harm in creating a genuine password here.

I can simplify this method to just SecureRandom.uuid if you like